src/js/constants.js

```diff
@@ -40,6 +40,16 @@ let exports = {
 
   PANOPTICLICK_DOMAINS: ["trackersimulator.org", "eviltracker.net"],
 
+  // Browser (modified during the build process)
+  BROWSER: "firefox",
```

Comment:
Note to self, update the release process to set `constants.BROWSER` appropriately.